src/App.css

```diff
@@ -0,0 +1,723 @@
+}
+
+@-moz-document url-prefix() {
+  .aside-text-effect {
+    line-height: 1;
+  }
+
+  .aside-text-effect:before {
+    line-height: 1;
+  }
+
+  .aside-text-effect:hover:before {
+    line-height: 1;
+  }
+}
+
+.aside-span-effect {
+  position: relative;
+  color: var(--light-pink-text-color);
+  -webkit-text-stroke: 2px #fff;
+}
+
+.aside-span-effect:before {
+  content: attr(data-fill);
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  color: #fff;
+  white-space: nowrap;
+  transition: 0.8s;
+  overflow: hidden;
+  clip-path: polygon(0 0, 0 1%, 100% 1%, 100% 0);
+}
+
+.aside-span-effect:hover:before {
+  height: 100%;
+  clip-path: polygon(0 0, 0 100%, 100% 100%, 100% 0);
+}
+
+/*------------------------------Footer---------------------------------*/
+
+.footer {
+  background: var(--footer-background-color);
+  grid-row: 3;
+  grid-column: 1/4;
+  display: flex;
+  flex-direction: column;
+  color: var(--background-color);
+  align-items: center;
+  justify-content: center;
+  background-color: #262626;
+}
+
+.footer-text {
+  margin-top: 3px;
+  font-size: calc(10 / 16 * 1rem) !important;
+}
+
+.footer-contacts {
+  margin-top: 8px;
+}
+
+.footer-contacts img {
+  margin-right: 5px;
+}
+
+.footer-signature {
+  margin: 5px;
+  font-size: calc(10 / 16 * 1rem) !important;
+  font-family: "Lucida Handwriting";
+}
+
+/*------------------------------Carousel of quotes---------------------------------*/
+
+.alice-carousel__dots {
+  display: none;
+}
+
+p.alice-carousel__prev-btn-item,
+p.alice-carousel__next-btn-item {
+  display: none;
+}
+
+div.alice-carousel__prev-btn,
+div.alice-carousel__next-btn {
+  display: none;
+}
+
+.alice-carousel ul li img {
+  display: flex;
+  height: 350px;
+  width: 50%;
+  object-fit: cover;
+}
+
+.slidertext {
+  width: 50%;
+  height: 100px;
+}
+
+blockquote {
+  text-align: end;
+  font-family: "Gilroy", sans-serif;
+}
+
+.quote1 {
+  font-family: inherit;
+  font-size: calc(20 / 16 * 1rem);
+  width: 380px;
+}
+
+.quote2 {
+  font-family: inherit;
+  font-size: calc(20 / 16 * 1rem);
+  width: 420px;
+}
+
+.quote3 {
+  font-family: inherit;
+  font-size: calc(20 / 16 * 1rem);
+  width: 660px;
+}
+
+.quote4,
+.quote5 {
+  font-family: inherit;
+  font-size: calc(20 / 16 * 1rem);
+  width: 580px;
+}
+
+.quote5 {
+  width: 470px;
+}
+
+.author {
+  font-family: "Gilroy", sans-serif;
+  font-size: calc(18 / 16 * 1rem);
+  border-right: 4px solid var(--light-pink-text-color);
+  padding-right: 10px;
+}
+
+.sidekick:before,
+.sidekick:after {
+  font-family: Calibri;
+  color: var(--light-pink-text-color);
+  font-size: calc(22 / 16 * 1rem);
+  margin-right: 5px;
+}
+.sidekick:before {
+  content: "\201c";
+  margin-right: 5px;
+  margin-left: 20px;
+}
+.sidekick:after {
+  content: "\201c";
+  margin-left: 3px;
+}
+
+/*------------------------------Carousel of images (in the section)---------------------------------*/
+
+.alice-carousel__wrapper {
+  width: 800px !important;
```

Comment:
This fixed 800px is the reason why the header carousel isn't adjustable. you can change this value to make it smaller when needed